_content/booklets/pathways-of-buddhist-thought-four-essays_nyanamoli.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Pathways of Buddhist Thought Four Essays"
+authors:
+  - nyanamoli
+external_url: "https://www.bps.lk/olib/wh/wh052_Nyanamoli_Pathways-Of-Buddhist-Thought--Four-Essays.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/10nsDXmod0VTuhqKjpj7fOKUrKc_-U9-e/view?usp=sharing"
+formats: [pdf]
+course: thought
```

Comment:
I know I put it in thought (based, no doubt, on the title), but I'm now second guessing that... Perhaps (with its focus on faith and understanding) this booklet more properly belongs in right `view`... What do you think?